tests/tracee_test.go

```diff
@@ -188,9 +188,21 @@ func TestTraceeSignatures(t *testing.T) {
 		os.RemoveAll(tempDir)
 	}()
 
-	for _, image := range []string{"tracee", "tracee-btfhub", "tracee-nocore"} {
-		for _, sigid := range []string{"TRC-3", "TRC-4", "TRC-9", "TRC-10", "TRC-11"} {
-			t.Run(fmt.Sprintf("%s/%s", image, sigid), func(t *testing.T) {
+	// Ubuntu 20.04 provided by GitHub Actions runner does not support CO:RE.
+	// Thus, we are running end-to-end signatures tests using tracee non CO:RE
+	// container image.
+
+	// FIXME Pass tracee container image flavor (tracee-nocore, tracee-core, etc.)
+	//       as input parameter to this test so we can set in the CI workflow
+	//       instead of hardcoding it here. The actual logic of the test should be
+	//       agnostic of tracee container flavor.
```

Comment:
Can we have an issue for this instead of a FIXME comment?

Reply:
I believe that to close #1191 and #1234 we must resolve these comments. And I'm actually going to jump right on it after we merge this PR.

Reply:
See #1411 that resolves FIXME and TODOs